kuka_agilus_support/urdf/kr6_r700-sixx_macro.xacro

```diff
@@ -5,118 +5,116 @@
   <xacro:include filename="$(find kuka_agilus_support)/urdf/kr_agilus_ros2_control.xacro" />
   <xacro:include filename="$(find kuka_agilus_support)/urdf/kr_agilus_transmission.xacro" />
 
-  <xacro:macro name="kuka_kr6_r700_sixx_robot" params="prefix client_ip client_port use_fake_hardware">
+  <xacro:macro name="kuka_kr6_r700-sixx_robot" params="prefix use_fake_hardware">
```

Comment:
While this name is technically not a name of a resource (in the ROS sense), I would also recommend against using dashes here.

Dashes in resource names are not legal (see [wiki/Names](http://wiki.ros.org/Names#Valid_Names), I don't believe there is a ROS 2 version of that page yet). Not using them in macro names would just make it easier for a user to guess or assume the spelling.